crypto/relic_build.sh

```diff
@@ -17,12 +17,21 @@ pushd "$DIR/relic/build"
 #
 GENERAL=(-DTIMER=CYCLE -DCHECK=OFF -DVERBS=OFF)
 LIBS=(-DSHLIB=OFF -DSTLIB=ON)
-COMP=(-DCOMP="-O3 -funroll-loops -fomit-frame-pointer -march=native -mtune=native")
+
+# "-march=native" is not supported on ARM
```

Comment:
Did compilation on ARM fail with gcc or clang ?

It seems that gcc [supports the flag](https://gcc.gnu.org/onlinedocs/gcc/ARM-Options.html#ARM-Options) (a node operator is also using gcc on ARM), but I couldn't find the same information for clang.

Reply:
Ah, yes, the flag is indeed [recently supported](https://www.phoronix.com/scan.php?page=news_item&px=GCC-8-march-native-ARM) on gcc. The OP in #809 almost certainly had issues with clang (because M1).

I've added 4cd200f, which I hope addresses the issue more narrowly, if at the cost of some complexity.